modules/flowable-engine-common/src/test/java/org/flowable/common/engine/impl/el/JsonNodeELResolverTest.java

```diff
@@ -0,0 +1,85 @@
+package org.flowable.common.engine.impl.el;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.flowable.common.engine.impl.javax.el.ELContext;
+import org.flowable.common.engine.impl.javax.el.ELResolver;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Per the project ide settings all static imports are listed first after the package name; thanks.

Reply:
Done